controllers/storagecluster/generate.go

```diff
@@ -45,11 +45,7 @@ func generateNameForCephBlockPool(initData *ocsv1.StorageCluster) string {
 }
 
 func generateDeviceClass(initData *ocsv1.StorageCluster) string {
```

Comment:
Maybe no need for this helper method anymore if you an directly use `initData.Status.DefaultCephDeviceClass`, but keeping it is find too if you want

Reply:
Yeah let's just remove this func